### DISCUSSION TITLE
How to render a space?

### DISCUSSION THREAD

**Discussion by monica-m-ps** [badges/shields#9552] [General] [answered]
Hi I am trying to use this on our site 
https://img.shields.io/badge/&nbsp;-iOS-blue?logo=Github

My requirement is to have the GiHub icon in one color and a text in the next block. I do not want any text in the same block as the GitHub icon

(GitHubicon)|(Some text here)

![image](https://github.com/badges/shields/assets/94380874/65e5a80e-fa16-4dee-ad85-3416aaa1ea90)

I am trying to render an empty space instead of the word test. https://img.shields.io/badge/test-iOS-blue?logo=Github)

Is that possible?

**Comment by chris48s** [comment, marked as answer]
I think this is the trick you are looking for: https://github.com/badges/shields/discussions/8193#discussioncomment-3149485

**Reply by monica-m-ps** [reply]
Thanks! It works!!
